barretenberg/cpp/src/barretenberg/stdlib/primitives/pairing_points.hpp

```diff
@@ -236,6 +259,7 @@ template <typename Builder> void read(uint8_t const*& it, PairingPoints<Builder>
     read(it, as.P0);
     read(it, as.P1);
     read(it, as.has_data);
+    read(it, as.tag);
```

Comment:
I'm honestly not sure if these serialization methods are even used - hopefully not - but either way should the tag be included here?